RELEASE_NOTES.md

```diff
@@ -1,5 +1,14 @@
 ﻿RELEASE NOTES
 
+**Version**: 1.0
+This versions requires:
+* Python for Unity 3.0.0
```

Comment:
You can remove the Python requirement, it could confuse users in thinking they need to install it themselves (it is pulled by Pacman as a dependency for the Shotgun package)